proto/ibc/lightclients/groth16/v1/groth16.proto

```diff
@@ -49,26 +51,18 @@ message Header {
   // StateTransitionProof is a serialized groth16 proof that the given state transition is valid.
   // If a proof with 0 bytes is provided, it is assumed to be a mock proof.
   bytes state_transition_proof = 1;
-
+  // PublicValues are bincode serialized public output from the SP1 program that get used as public witness for the verifier.
+  bytes public_values = 2;
   // TrustedHeight is the last verified height of the rollup. This is used to retrieve the previous
   // state root with which the proof is verified against.
-  int64 trusted_height = 2;
-
-  // TrustedCelestiaHeaderHash is the last verified Celestia header hash.
-  bytes trusted_celestia_header_hash = 3;
-
-  // NewStateRoot is the new state root of the rollup after the state transition has been applied.
-  bytes new_state_root = 4;
-  // NewHeight is the new height of the rollup after the state transition has been applied.
-  int64 new_height = 5;
-  // NewCelestiaHeaderHash is the new Celestia header hash after the state transition has been applied.
-  bytes new_celestia_header_hash = 6;
+  int64 trusted_height = 3;
 
-  // TODO: This is provided by the user at the moment but we can't trust them
-  // with this data. We need to get all the data roots from the
-  // the store.
-  repeated bytes data_roots = 7;
 
+  // NewestStateRoot is the computed state root of the EVM roll-up after
+	// processing blocks from oldest header hash to newest_header_hash
```

Comment:
The indentation here looks incorrect. I think it currently uses tabs but the rest of file uses spaces. I'm surprised proto-lint didn't catch this.

```suggestion
  // processing blocks from oldest header hash to newest_header_hash
```